tests/forms/test_address_upload.py

```diff
@@ -0,0 +1,170 @@
+import pytest
+
+from pretix.base.models import (
+    User, Team,
+)
+from django.core.files.uploadedfile import SimpleUploadedFile
+from pretix_eth.models import WalletAddress
+
+
+@pytest.fixture
+def env(event):
```

Comment:
The name of this fixture feels disconnected from what it returns.  Maybe call it something like `wallet_address`?